### ISSUE TITLE
New v1.2 release?

### ISSUE THREAD

**Issue by carlosperate** [google/bloaty#334]
First of all, thanks for creating this very useful tool, it's really appreciated!

It's been a couple of years and almost 250 commits since the v1.1 release. Would it be possible to make a new, tagged, v1.2 release?

**Comment by chenrui333** [comment]
any update on this thread? Now facing the issue upgrading re2 (as abseil requires c++14 or higher to build)

**Comment by louwers** [comment]
Getting 

> bloaty: Data is in new DWARF format we don't understand

with the released version but building from source works fine...

**Comment by carlocab** [comment]
CC @haberman 

It'd be nice to cut a new release tag so that various distros that package bloaty can ship a new version: https://repology.org/project/bloaty/versions

**Comment by marciogranzotto** [comment]
yeah, just installed this with homebrew and discovered I have to build from source to have DWARF 5 support... at least the github action `build` could upload the binary as an artifact
